avm/src/daemon/daemon.rs

```diff
@@ -19,7 +27,11 @@ use tokio::time::{sleep, Duration};
 
 use crate::daemon::dns::DNS;
 use crate::daemon::lrh::LogRendezvousHash;
-use crate::vm::run::run_agz_b64;
+use crate::make_server;
+use crate::vm::http::{HttpConfig, HttpType, HttpsConfig};
+use crate::vm::run::run;
+
+pub static SECRET_STRING: OnceCell<Option<String>> = OnceCell::new();
```

Comment:
nit should we maybe call this `CLUSTER_TOKEN`? `SECRET_STRING` is accurate, but it also doesn't say much about the actual use case of the secret?

Reply:
So `cluster_token` is very confusing with `deploy_token`, which is why I changed that. It also does have multiple uses; it's not just for the inter-cluster-communication but verifying `deploy` to cluster communication, and potentially could be used in other ways, too?

Reply:
Actually, I do have a question on that front. What are we using the `deploy_token` for that couldn't be handled here? I wouldn't mind calling it the `cluster_token` if we can just eliminate the `deploy_token`?

Reply:
I don't think haven two tokens is confusing. They have different use cases, but more of a nit. `deploy` to cluster communication still falls under correct usage or proper naming for something called cluster token to me. The deploy token is given from `deploy` to the `daemon` and then back (since `deploy` is stateless) to the provide the deployment configuration (credentials, region, vmType) to `deploy` when it needs to add or remove more VMs to the cluster

Reply:
And the deploy token was deemed unusable for authentication since it can change during an upgrade

Reply:
Maybe `CLUSTER_SECRET`?

Reply:
I like `CLUSTER_SECRET`

Reply:
> I don't think having two tokens is confusing? They have different use cases, but more of a nit. `deploy` to cluster communication still falls under correct usage or proper naming for something called cluster token to me. The deploy token is given from `deploy` to the `daemon` and then back (since `deploy` is stateless) to the provide the deployment configuration (credentials, region, vmType) to `deploy` when it needs to add or remove VMs from the cluster

I forgot that the `deploy_token` was a signed, encrypted payload of stuff to make upgrades possible.

And in that case keeping `secret` in the name of this variable makes sense because it's not encrypting anything.

Reply:
Yeah, I also prefer `CLUSTER_SECRET` to `CLUSTER_TOKEN` for that reason